### PR TITLE
Updated tests related to changes in oraganization-id options.

### PR DESCRIPTION
**Pull request by svtkachenko**
Updated some cli/factory methods -- added new cli options.
Added tests to check cases with multiple organizations.

Test results:
```python
py.test -v -k "HostGroupTestCase and test_positive_create_with_multiple_entities or test_positive_create_with_orgs" tests/foreman/cli/test_hostgroup.py                                                                                                                          
===================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.12, pytest-2.9.2, py-1.4.31, pluggy-0.3.1 -- /home/qui/code/robottelo/venv2/bin/python2.7
cachedir: .cache
rootdir: /home/qui/code/robottelo, inifile: 
plugins: cov-2.3.1, xdist-1.14
collected 21 items 

tests/foreman/cli/test_hostgroup.py::HostGroupTestCase::test_positive_create_with_multiple_entities <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_hostgroup.py::HostGroupTestCase::test_positive_create_with_orgs PASSED
tests/foreman/cli/test_hostgroup.py::HostGroupTestCase::test_positive_create_with_orgs_and_lce PASSED

=============== 18 tests deselected by '-kHostGroupTestCase and test_positive_create_with_multiple_entities or test_positive_create_with_orgs' ================
========================================================== 3 passed, 18 deselected in 193.18 seconds ==========================================================
```

### CODE REVIEW

File: tests/foreman/cli/test_hostgroup.py [hunk 1]
```diff
@@ -128,6 +135,24 @@ def test_positive_create_with_org(self):
...
         self.assertIn(org['name'], hostgroup['organizations'])
 
+    @tier1
+    def test_positive_create_with_orgs(self):
+        """Check if hostgroup with multiple organizations can be created
+
+        @id: 32be4630-0032-4f5f-89d4-44f8d05fe585
+
+        @Assert: Hostgroup is created and has both new organizations assigned
+
```
Comment (oshtaier) [inline tests/foreman/cli/test_hostgroup.py]: Redundant line of code

File: tests/foreman/cli/test_hostgroup.py [hunk 2]
```diff
@@ -204,16 +229,38 @@ def test_positive_create_with_lifecycle_environment(self):
...
     @tier1
+    def test_positive_create_with_orgs_and_lce(self):
+        """Check if hostgroup with multiple organizations can be created
+        if one of them is associated with lifecycle environment
+
+        @id: 32be4630-0032-4f5f-89d4-44f8d05fe585
+
+        @Assert: Hostgroup is created, has both new organizations assigned
+        and has lifecycle env assigned
+
```
Comment (oshtaier) [inline tests/foreman/cli/test_hostgroup.py]: Redundant line of code

**Comment by oshtaier** [comment]
ACK pending nitpicks

**Comment by sghai** [comment]
ACK pending on @oshtaier comments.

**Comment by oshtaier** [comment]
ACK pending test results

**Comment by oshtaier** [comment]
ACK

**Pull request merged**